src/Workspaces/VisualBasic/Portable/Utilities/SpeculationAnalyzer.vb

```diff
@@ -431,18 +431,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Utilities
             Return forEachControlVariable IsNot Nothing AndAlso forEachControlVariable.IsTypeInferred(Me.OriginalSemanticModel)
         End Function
 
-        Protected Overrides Function IsInvocableExpression(node As SyntaxNode) As Boolean
-            If node.IsKind(SyntaxKind.InvocationExpression) OrElse node.IsKind(SyntaxKind.ObjectCreationExpression) Then
-                Return True
-            End If
-
-            If node.IsKind(SyntaxKind.SimpleMemberAccessExpression) AndAlso
-                Not node.IsParentKind(SyntaxKind.InvocationExpression) AndAlso
-                Not node.IsParentKind(SyntaxKind.ObjectCreationExpression) Then
-                Return True
-            End If
-
-            Return False
+        Protected Overrides Function ExpressionMightReferenceMember(node As SyntaxNode) As Boolean
+            Return node.IsKind(SyntaxKind.InvocationExpression) OrElse
+                node.IsKind(SyntaxKind.SimpleMemberAccessExpression)
```

Comment:
C# checked for member-binding.  This code doesn't.  Is that intentional?

Reply:
The VB `SyntaxKind` doesn't have a MemberBinding kind.